src/main/java/uk/gov/hmcts/ccd/domain/model/draft/DraftResponse.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.ccd.domain.model.draft;
+
+import lombok.ToString;
+
+import java.time.LocalDateTime;
+
+@ToString
+public class DraftResponse {
+
+    private String id;
+
+    private CaseDraft document;
+
+    private String type;
+
+    private LocalDateTime created;
```

Comment:
in the Draft class we are using ZonedDateTime for these props. Here a LocalDataTime. Is this ok?